### PR TITLE
Blog: OpenSSL 1.0.2m announcement

### PR DESCRIPTION
**Pull request by rvagg**
Ref: https://github.com/nodejs/Release/issues/271

### CODE REVIEW

File: locale/en/blog/vulnerability/openssl-november-2017.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+>
+> If an X.509 certificate has a malformed IPAddressFamily extension, OpenSSL could do a one-byte buffer overread. The most likely result would be an erroneous display of the certificate in text format.
+> 
+> As this is a low severity fix, no release is being made. The fix can be found in the source repository (1.0.2, 1.1.0, and master branches); see https://github.com/openssl/openssl/pull/4276. This bug has been present since 2006.
+
+At this stage, due to embargo, it is uncertain what the nature of the "moderate" severity fix is, nor what impact it will have on Node.js users, if any. We will proceed as follows:
+
+Within approximately 24 hours of the OpenSSL 1.0.2m release, our crypto team will make an impact assessment for Node.js users. This information _may_ vary depending for the different active release lines and will be posted here.
+
+As part of that impact assessment we will announce our release plans for each of the active release lines to take into account any impact. **Please be prepared for the possibility of important updates to Node.js 4 "Argon", Node.js 6 "Boron", Node.js 8 "Carbon" and Node.js 9 (Current) as soon as Friday, the 4th of November, 2017**.
```
Comment (MylesBorins) [inline locale/en/blog/vulnerability/openssl-november-2017.md]: 3rd of November

**Review by MylesBorins — APPROVED** [review]
Lgtm with small nit for date

File: locale/en/blog/vulnerability/openssl-november-2017.md (hunk 1 above)
Comment (rvagg) [reply locale/en/blog/vulnerability/openssl-november-2017.md]: :thumbsup:

**Comment by rvagg** [comment]
merged @ b387bf9aac0b

**Pull request closed without merging**